lib/plug/static.ex

```diff
@@ -77,6 +77,11 @@ defmodule Plug.Static do
 
     * `:headers` - other headers to be set when serving static assets.
 
+    * `:types` - custom MIME type mapping. As a map with filename as key
```

Comment:
I used `mime_types` at first, but I changed because I thought that in a config, `types: %{"foo" => "application/json"}` was quite self explanatory.